src/app/shared/alert/alert.component.css

```diff
@@ -0,0 +1,4 @@
+.mat-card {
+  padding: 15px;
+  padding-right: 35px;
```

Comment:
comment that the padding-right came from bootstrap's alert-dismissable class and the padding came from bootstrap's alert class.   Double-check whether alert and alert-dismissable classes are doing anything on the mat-card and remove them if they aren't